facebook/__init__.py

```diff
@@ -235,7 +243,7 @@ def request(
             # or it does not need `access_token`.
             if post_args and "access_token" not in post_args:
                 post_args["access_token"] = self.access_token
-            elif "access_token" not in args:
+            elif args and "access_token" not in args:
```

Comment:
This is probably a different bug... I had a test that was failing before, though when I revert this now that same test isn't failing, so I'm not sure what changed. Your call, do you want this reverted or included?